docker-run-fetch-flow-definitions.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME=adss
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --profile-cpu)
+            PROFILE_CPU=true
```

Comment:
Is the idea to make this available on all the containers? I'm fine with that if so - I don't think it's necessary for us to be able to profile this, but if it improves consistency I'm fine with it.

Reply:
Yes, it's really just for consistency. There are a few commands later in this script which start to diverge from the other run scripts if we drop it.